packages/gql-executor/tracer.ts

```diff
@@ -1,14 +1,17 @@
 import tracer from 'dd-trace'
-import PROD from '../server/PROD'
-if (PROD) {
-  tracer.init()
-}
 
-tracer.use('graphql', {
-  hooks: {
-    execute: (span, args) => {
-      span.setTag('viewerId', args?.contextValue?.authToken?.sub ?? "null")
+if (process.env.DD_TRACE_ENABLED === 'true') {
+  tracer.init({
+    plugins: false,
+    debug: false
+  })
+  tracer.use('graphql', {
+    enabled: true,
```

Comment:
Since the resolvers in the schema are patched for graphql-jit, this plugin might be broken.